File: Development/PS/if/2.js
const fs = require("fs");
const input = fs.readFileSync("/dev/stdin").toString().split(" ");

const score = Number(input[0]);

if (100 >= score && score >= 90) {
  console.log("A");
} else if (89 >= score && score >= 80) {
  console.log("B");
} else if (79 >= score && score >= 70) {
  console.log("C");
} else if (69 >= score && score >= 60) {
  console.log("D");
} else {
  console.log("F");
}

File: Development/PS/baekjoon/Python/for/2.py
import sys

input()  # 인풋함수 선언으로 input 기능 활성화
s = list(sum(map(int, n.split()))
         for n in sys.stdin)  # list 컴프리헨션으로 n 변수 활성화 및 합계값 list


for n in s:   # itrable 된 s print
    print(n)
